js/blockchaincom.js

```diff
@@ -876,7 +876,7 @@ module.exports = class blockchaincom extends Exchange {
             account['total'] = this.safeString (entry, 'balance');
             result[code] = account;
         }
-        return this.parseBalance (result);
+        return result;
```

Comment:
It isn't really fixing the bug, introducing a new one – incomplete balances if we remove parseBalance from here. Instead of removing it we should simply replace it with string-based safeBalance.